eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -20,26 +20,25 @@ import { ETheme } from "@/themes";
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 type UserBooking = BookingsMeta & Pick<Customer, "certificateExpiration">;
 
-interface Props extends Slot<"id"> {
+interface Props extends SlotInterface {
   userBookings: UserBooking[];
-  absentees?: string[];
 }
 
 // mark attendees
 const AttendanceCard: React.FC<Props> = ({
   date,
-  durations,
   categories,
   userBookings,
-  absentees,
+  attendance,
+  intervals,
   type,
   id,
 }) => {
   const classes = useStyles();
 
   // const { t } = useTranslation();
   // convert timestamp to luxon for easier processing
-  const luxonStart = fb2Luxon(date);
+  // const luxonStart = fb2Luxon(date);
 
   // convert durations to number values
   const durationNumbers = durations.map((duration) => Number(duration));
```

Comment:
Yeah, I think you we're asking about this. Yes, this will absolutely go and get replaced by intervals.